_components/customisation.md

```diff
@@ -0,0 +1,199 @@
+
+### Customise A Brand
+
+If your project has access to a component's Sass API (see the [manual build tutorial](/docs/tutorials/manual-build/#selecting-a-brand)) it is possible to customise your chosen brand. For example, at the time of writing, the [`oTypographySetFont`](https://registry.origami.ft.com/components/o-typography@6.4.5/sassdoc?brand=master#mixin-otypographysetfont) mixin can be used to customise component fonts. This customises the font used by all components including in your project.
+
+<aside>
+Not all components support brand customisation but the Origami team are happy to add support as needed. Ask the core Origami team for new features in the <a href="https://financialtimes.slack.com/messages/{{site.data.contact.slack}}">#{{site.data.contact.slack}}</a> Slack channel.
+</aside>
+
+<pre><code class="o-syntax-highlight--scss">// Incomplete example for demo purposes only
+
+// Set the brand.
+$o-brand: 'whitelabel';
+
+// Customise: Tell the o-typography component to use our custom
+// font, it will affect all other components which
+// as they are built using o-typography.
+@include oTypographySetFont(
+    $type: 'sans',
+    $family: 'Comic Sans MS', sans-serif
+);
+
+// Customise: Tell the o-header-services we want hot pink
+// in the navigation.
+@include oHeaderServicesCustomize((
+	'nav-hover-background': hotpink
+));
+</code></pre>
+
+
+<figure>
+	<img alt="" src="/assets/images/component-customisation/whitelabel-customisation.png" />
+	<figcaption>
+        An example project which builds on the whitelabel brand. The font has been customised with `o-typography` Sass (oooh! Comic Sans). It affects all components which use `o-typography`, including the other components on the page, `o-table` and `o-header-services`. `o-header-services` has also been customised using Sass to have a unique highlight colour (oooh! hotpink!)
+	</figcaption>
+</figure>
+
+This is great to create a distinct brand, where a customisation should affect all instances of a component across a project.
+
+If your project is based on the master or internal brand, however, it is more likely that you want to create new variants of a component to use alongside existing ones. For example to output an additional banner style to support a new marketing campaign.
+
+## Themes/Variants
+
+If your project has access to a component's Sass (see the [manual build tutorial](/docs/tutorials/manual-build/#selecting-a-brand)) your project may use Sass mixins to customise components by outputting a new variant of that component. A component variant is an instance of a component with modified appearance or functionality.
+
+For instance, at the time of writing, the `o-banner` component has an [`oBannerAddTheme` mixin](https://registry.origami.ft.com/components/o-banner@3.4.2/sassdoc?brand=master#mixin-obanneraddtheme) which provides options for creating a banner theme with customised colour and background image:
```

Comment:
I've now come to realise that we have o-brand use the term ["Variant"](https://registry.origami.ft.com/components/o-brand@4.0.0/readme#variant) and has defined what it means, but we have some component mixins use the term "theme" to mean the same thing.

We should plan some work to migrate to using only one of these terms.

Reply:
As discussed: 

A theme is a kind of variant, at least in my head. A table with alternating row stripe colours is a variant of the default table with no row stripes. A theme could be applied to the row stripe variant to change the stripe colours. I agree that it's confusing to use both terms and we should switch to using one. Although theme is a much more familiar term it doesn't work for all variants of a component, especially variants with unique functionality as well as appearance. I'll update these docs to use "variant" and delete mentions of "theme" outside the Sass mixins. In the future we might want to rename the mixins.